stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -400,6 +408,8 @@ public void start() throws BundleException {
       bundle.start();
     }
 
+    System.out.println(STARTED_MESSAGE);
```

Comment:
maybe use `logger.info` instead of `system.out`?

Reply:
This class already uses `System.out` to print package activation messages... so I thought this would fix the pattern.